OpenSim/Common/Test/CMakeLists.txt

```diff
@@ -15,5 +15,5 @@ OpenSimAddTests(
     TESTPROGRAMS ${TEST_PROGS}
     DATAFILES ${TEST_FILES} ${C3D_TEST_FILES} ${TRC_TEST_FILES} 
               ${MOT_TEST_FILES}
-    LINKLIBS ${BTK_LIBRARIES} osimCommon ${SIMTK_ALL_LIBS} 
+    LINKLIBS ${BTK_LIBRARIES} osimCommon osimSimulation ${SIMTK_ALL_LIBS} 
```

Comment:
I don't think it is sufficient to check the column labels without simulating (e.g., `initSystem()` was reconnecting the Inputs and assigning default aliases—took me a long time to find that). I'll move the test to the Simulation Tests folder.